source/code/plugins/scom_configuration.rb

```diff
@@ -0,0 +1,100 @@
+module SCOM
+
+  class Configuration
+    require 'openssl'
+    require 'uri'
+        
+    require_relative 'oms_configuration'
+    require_relative 'omslog'
+        
+    @@configuration_loaded = false
+    @@scom_conf_path = '/etc/opt/microsoft/omsagent/scom/conf/omsadmin.conf'
```

Comment:
Remember: if you change the workspace-id-placeholder in omsadmin.sh from "scom" to "SCOM", you will have to rename these paths.